viewer/packages/cad-geometry-loaders/src/CadModelUpdateHandler.ts

```diff
@@ -138,6 +141,9 @@ export class CadModelUpdateHandler {
   addModel(model: CadNode): void {
     this._modelStateHandler.addModel(model.cadModelMetadata.modelIdentifier);
     this._modelSubject.next({ model, operation: 'add' });
+    model.nodeAppearanceProvider.on('prioritizedAreasChanged', () => this.updatePrioritizedAreas());
+    // Ensure there is at least one call in the line for the model to load
+    this.updatePrioritizedAreas();
```

Comment:
I experienced that without this, the model is not loaded at all, probably because rxjs requires there to be a signal sent for each input subject to trigger load. Is there a better way of fixing this?

Reply:
Hmm, not sure I understand. Without this the model isn't loaded at all? How has this worked before? Looks quite dodgy - wouldn't `_modelSubject.next()` trigger a change so that data is loaded?

Reply:
It is documented here: https://rxjs.dev/api/index/function/combineLatest
> combineLatest will actually wait for all input Observables to emit at least once
So it seems we need to emit at least once to make it load. It fits with my experience.
Anyway, I found a cleaner way, using `startWith` to create an initial value.

Reply:
Great, startWith is bette r:)